plugin/trino-hive/src/test/java/io/trino/plugin/hive/parquet/AbstractTestParquetReader.java

```diff
@@ -2164,4 +2171,38 @@ private static SqlDate intToSqlDate(Integer input)
         }
         return new SqlDate(input);
     }
+
+    private static CustomParquetWriter getParquetWriter(MessageType schema, File outputFile)
+            throws IOException
+    {
+        Path path = new Path(outputFile.toURI().toString());
+        return new CustomParquetWriter(path, schema, false, CompressionCodecName.SNAPPY);
+    }
+
+    @Test
```

Comment:
Test method should be placed above all the private methods just after `testMapMaxReadBytes`